core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -56,8 +51,12 @@ public SetupWizard(Jenkins j) throws IOException {
             HudsonPrivateSecurityRealm securityRealm = new HudsonPrivateSecurityRealm(false, false, null);
             j.setSecurityRealm(securityRealm);
             String randomUUID = UUID.randomUUID().toString().replace("-", "").toLowerCase(Locale.ENGLISH);
-            admin = securityRealm.createAccount(SetupWizard.initialSetupAdminUserName, randomUUID);
-            admin.addProperty(new SetupWizard.AuthenticationKey(randomUUID));
+            
+            // create an admin user
+            securityRealm.createAccount(SetupWizard.initialSetupAdminUserName, randomUUID);
+            
+            // JENKINS-33599 - write to a file in the jenkins home directory
```

Comment:
The user who does yum install would still be able to read this file as they are root or can sudo....
(ok unless they can sudo yum but not sudo bash - but anyone that has that setup is braindead!)

If the file is readable by anyone logged into the system then that seems like an issue to me.
besides - I beleive the jenkins HOME will be 700 jenkins:jenkins anyway - so this file should be explicitly set for the multi-user not using installers case!